hw/ip/csrng/lint/csrng.waiver

```diff
@@ -7,3 +7,5 @@
 waive -rules {ONE_BIT_MEM_WIDTH} -location {prim_arbiter_ppc.sv} -regexp {.*has word width which is single bit wide.*} \
       -comment "Usage case specific to CSRNG and how the arbiter is used."
 
+waive -rules {WIDTH} -location {aes_cipher_core.sv aes_cipher_control_fsm.sv aes_sbox.sv aes_key_expand.sv} -regexp {*TODO*} \
+      -comment "CSRNG intentionally uses an unmasked AES implementation."
```

Comment:
@vogelpi I ran ascentlint for csrng and this branch, and got the following message:
ERROR, -rules argument WIDTH is not a rule or category name
  ... while executing : waive -rules WIDTH -location {aes_cipher_core.sv aes_cipher_control_fsm.sv aes_sbox.sv aes_key_expand.sv} -regexp *TODO* -comment {CSRNG intentionally uses an unmasked AES implementation.}

Reply:
Maybe just omit this waiver for now and fix it later once it shows up on the dashboard?

Reply:
Yes, that's easier, I've now removed that change and will add a lint waiver once the error shows up in the dashboard.